### ISSUE TITLE
vim visual selection -> OS-level copy: loses last character

### ISSUE THREAD

**Issue by eavesjump** [googlecolab/colabtools#2748]
I'm on macOS, using both Chrome and Firefox.

Selecting text using vim visual mode, then pressing Cmd+C to copy the selection, you end up only copying the selection minus the last character in the selection. This in spite of vim commands (including eg vim's copy) working on the actual, full selection as expected. For example, if the vim selection is `foo`, Cmd+C only copies `fo`.

(This can be surprisingly pernicious. Just spent 30m figuring this out because it combined with a github personal access token bug that made both bugs really hard to figure out. It's very hard to see that your random password is missing a character when the "wrong password" error message is missing/buggy.)

**Comment by craigcitro** [comment]
Yes, this is pretty unfortunate; we'll see if there's anything we can do on our side. (We're using a plugin for vim keyboard mode, so this may be out of our hands.)
